test/NuGet.Core.Tests/NuGet.Configuration.Test/PackageSourceTests.cs

```diff
@@ -59,7 +59,7 @@ public void AsSourceItem_WorksCorrectly()
         }
 
         [Fact]
-        void CalculatedMembers_ForHttpsSource_HasExpectedValues()
+        public void CalculatedMembers_ForHttpsSource_HasExpectedValues()
```

Comment:
xunit ship their own analyzers, and I thought they warn about non-public methods marked as tests.   Are those analyzers also disabled?